trussworks/react-uswds
# docs(storybook): Add templates for authentication pages

# Summary

Implements the USWDS [authentication page](https://designsystem.digital.gov/templates/authentication-pages/) templates in Storybook.

## Related Issues or PRs

Closes #1102 

## How To Test

* `yarn lint` and verify no errors appear.
* In storybook, view the three files in "Authentication pages" folder located in the "Templates" section, and verify that they match the USWDS implementations for
  * [Create account](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/release-2.11.2/components/preview/layout--create-account.html)
  * [Sign in](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/release-2.11.2/components/preview/layout--sign-in.html)
  * [Multiple sign in options](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/release-2.11.2/components/preview/layout--sign-in-multiple.html)
